packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -66,27 +68,15 @@ export default class Python extends Target {
 // # CODE GENERATOR #
 // ##################
 
-const PYTHON_BUILTIN_TYPES = ['bool', 'str', 'None'];
-
-const PYTHON_KEYWORDS = [
-  'False', 'None', 'True', 'and', 'as', 'assert', 'async', 'await', 'break', 'class',
-  'continue', 'def', 'del', 'elif', 'else', 'except', 'finally', 'for', 'from',
-  'global', 'if', 'import', 'in', 'is', 'lambda', 'nonlocal', 'not', 'or', 'pass',
-  'raise', 'return', 'try', 'while', 'with', 'yield'
-];
+interface EmitContext extends NamingContext {
+  /** @deprecated The TypeResolver */
```

Comment:
I'm marking this deprecated because *in fine*, I intend on getting rid of the `TypeResolver` class as a whole.  It's grown overly complex and I believe its purposes can be achieved in more reliable (expressive?) ways with the information present in the assemblies.

But this is tricky and this PR is already large enough that I didn't want to go forward with it at this moment.